checkov/terraform/checks/resource/aws/SQSQueuePolicyAnyPrincipal.py

```diff
@@ -19,8 +19,11 @@ def scan_resource_conf(self, conf):
         if conf_policy:
             if isinstance(conf_policy[0], dict):
                 policy = Policy(conf_policy[0])
-                if policy.is_internet_accessible():
-                    return CheckResult.FAILED
+                try:
+                    if policy.is_internet_accessible():
```

Comment:
```suggestion
                try:
                    policy = Policy(conf_policy[0])
                    if policy.is_internet_accessible():
```
does it make sense to also move the initial part inside the try... except block?